nubia/internal/context.py

```diff
@@ -22,6 +25,7 @@ def __init__(self):
         self._testing = None
         self._registry = None
         self._args = {}
+        self._tier = ""
```

Comment:
Please don't expose the tier via the context. It's specific to LogDevice use case and we shall avoid it in Nubia.

Reply:
Ah ok. If this is application specific, I assume you want to factor it out of nubia entirely, correct? If that is the case then are you wanting the default prompt to just be `USERNAME> `?